vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/ConsumableResourceChecker.java

```diff
@@ -104,7 +104,7 @@ public Optional<String> unavailable() {
         while (--i >= 0) {
           resourceBrokers.get(i).second().release(workflow, workflowVersion, vidarrId);
         }
-        executor.schedule(this, 5, TimeUnit.MINUTES);
+        executor.schedule(this, 2, TimeUnit.MINUTES);
```

Comment:
I think this is too short of an interval - that's more than doubling the frequency of checking/load on the Vidarr processor. I'd prefer 4 minutes if you want to decrease the whole thing.
Or, since the goal of decreasing this is to decrease the amount of time spent waiting if it fails to acquire the PriorityByWorkflow resource (because there's something higher priority in WAITING), perhaps it could be rescheduled to a shorter time only if it's the PriorityByWorkflow resource that fails the check? That way it would only increase the frequency in some cases, instead of across the board.
(But I think we could also leave it at 5 minutes and see if it significantly increases time-to-launch for workflow runs - from what I calculated, leaving it as-is would result in single-digit minutes spent waiting with an idle max-in-flight slot. @grapigeau @mlaszloffy thoughts?)

Reply:
I think it would be relatively straightforward to check if the reason it's rescheduled contains a message like "There are workflows currently running with higher priority." and have that result in a different (shorter) wait time. I can set that to 4 minutes and have the default be 5?

Reply:
I'm not sure why we're changing this value, but - checking String contents to decide on behaviour is liable to break with further maintenance - could the different refresh windows move to be a property of each ConsumableResource type rather than establishing what might end up being a large decision block as we add more ConsumableResource types each with custom behaviour here?